### PR TITLE
Remove default laser direction parameters

### PR DESCRIPTION
**Pull request by roelof-groenewald**
The default propagation direction and polarization for lasers were removed and the laser antenna normal direction was made optional.
The current PR is in response to [this](https://github.com/ECP-WarpX/WarpX/pull/3477#discussion_r1003555620) discussion.

### CODE REVIEW

File: PICMI_Python/lasers.py [hunk 1]
```diff
@@ -83,10 +83,10 @@ class PICMI_GaussianLaser(_ClassWithInit):
         Optional name of the laser
     """
     def __init__(self, wavelength, waist, duration,
+                 propagation_direction,
+                 polarization_direction,
                  focal_position = [0., 0., 0.],
```
Comment (dpgrote) [inline PICMI_Python/lasers.py:88]: While you are at it, might as well remove these default values as well.

File: PICMI_Python/lasers.py [hunk 2]
```diff
@@ -221,10 +221,10 @@ class PICMI_LaserAntenna(_ClassWithInit):
     position: vector of strings
         Position of antenna launching the laser [m]
 
-    normal_vector: vector of strings
+    normal_vector: vector of strings, optional
         Vector normal to antenna plane [1]
```
Comment (dpgrote) [inline PICMI_Python/lasers.py:225]: Perhaps add a comment that this defaults to the laser direction of propagation.

File: PICMI_Python/setup.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
     install_requires = [ line.strip('\n') for line in f.readlines() ]
 
 setup(name = 'picmistandard',
-      version = '0.0.21',
+      version = '0.0.22',
```
Comment (dpgrote) [inline PICMI_Python/setup.py:12]: The version update should be done in a commit separately from this PR. See `PICMI_Python/README.md` for how to update the version.

Comment (roelof-groenewald) [reply PICMI_Python/setup.py]: I saw the README direction saying "it is ok to push a version update directly to the main branch", but I didn't understand that it **should** be done on the main branch. Now I know for next time :)

**Review by dpgrote — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**